pkg/utils/accessLogUtils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/siglens/siglens/pkg/common/dtypeutils"
+	log "github.com/sirupsen/logrus"
+)
+
+func AddAccessLogEntry(data dtypeutils.AccessLogData, fileName string) {
+	logFile, err := os.OpenFile(fileName, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if err != nil {
+		log.Errorf("Unable to write to access.log file, err=%v", err)
+	}
+	defer logFile.Close()
+	// Do not log websocket connections
+	if data.StatusCode == 101 {
+		return
+	}
+	// Do not log internal search requests for trace data
+	if strings.TrimSpace(data.URI) == "http:///" || strings.TrimSpace(data.URI) == "https:///" {
```

Comment:
combine these two if loops  , we do not want to log queries with "http:///" AND indexName=traces

Reply:
Have made the changes